examples/wbnf.wbnf

```diff
@@ -40,10 +40,8 @@ RE      -> /{
                )(?: (?:[+*?]|\{\d+,?\d?\}) \?? )?
              )+
            };
-REF     -> "%" IDENT ("=" default=STR)?;
-// Special
 pragma  -> (
                 import=(".import" path=((".."|"."|[a-zA-Z0-9.:]+):,"/"))
-           );
+REF     -> "%" IDENT ("=" default=STR)?;
```

Comment:
revert this change, looks like a bad merge

Reply:
reverted